bcmi/src/app/footer/footer.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, OnInit, Input, ViewEncapsulation } from '@angular/core';
+import { Footer } from '@app/models/content/footer';
+import { Api } from '@app/services/api';
+
+@Component({
+  selector: 'app-footer',
+  templateUrl: './footer.component.html',
+  styleUrls: ['./footer.component.scss'],
+  encapsulation: ViewEncapsulation.None,
+})
+
+export class FooterComponent implements OnInit {
+
+  @Input() footerData: Footer;
+  @Input() navigation: any;
+  hostname: string;
+  constructor( private api: Api) { }
+
+  ngOnInit() {
+    this.hostname = this.api.hostnameNRPTI;
+    window.scrollTo(0, 0);
```

Comment:
Do we want the footer component to call `window.scrollTo()`? The page component already does that, and we can add the same logic to the home page if it doesn't exist already.